index.js

```diff
@@ -95,6 +95,10 @@ SonicBoom.prototype.end = function () {
   actualClose(this)
 }
 
+SonicBoom.prototype.flush = function () {
+  actualWrite(this)
+}
```

Comment:
This needs to be protected. If we are already writing, we should not be writing more.